frontend/src/main.ts

```diff
@@ -1,6 +1,29 @@
 import { createApp } from 'vue';
+import { createRouter, createWebHashHistory } from 'vue-router';
 import App from './App.vue';
 
 import './assets/main.css';
+//routes
+import GuidePageVue from './routes/guide/GuidePage.vue';
+import IndexPageVue from './routes/index/IndexPage.vue';
+import QuestionPageVue from './routes/question/QuestionPage.vue';
+import ResultPageVue from './routes/result/ResultPage.vue';
+import SummaryPageVue from './routes/summary/SummaryPage.vue';
 
-createApp(App).mount('#app');
+const routes = [
+  { path: '/', component: IndexPageVue },
+  { path: '/guide/:nr', component: GuidePageVue },
+  { path: '/question/:nr', component: QuestionPageVue },
+  { path: '/summary', component: SummaryPageVue },
+  { path: '/result', component: ResultPageVue },
+];
+
+const router = createRouter({
+  history: createWebHashHistory(),
```

Comment:
Pokud guide a questions nebudou mít svoji url, stopro bych byl pro `createWebHistory()`, ten hash tag v url se mi nelíbí a taky píší, že tento `createWebHashHistory()` má dopad na SEO a ten asi budem potřebovat mít na top.

Reply:
to je pravda ale na druhou stranu to nema vliv na server. muzeme to prehosit pozdeji az se nastavi backend aby routoval 404 na index